docs/readthedocs/io/basiciousage.rst

```diff
@@ -9,4 +9,19 @@ Therefore, the I/O-package is structured as highly modular.
    :maxdepth: 2
 
    influxdb
-   csvfiles
\ No newline at end of file
+   csvfiles
+
+
+
+Data sink structure
+===================
+.. figure:: ../_static/figures/uml/DataSinkClassDiagram.png
+   :align: center
+   :alt: Class diagram of data sink classes
```

Comment:
If you add `   :width: 650` like in `csvfiles`, the images become clickable and open in full size. That'd be nice!

Pls adapt for all new images